package.json

```diff
@@ -84,10 +89,12 @@
           "type": "string",
           "enum": [
             "babylon",
-            "flow"
+            "flow",
+            "typescript",
+            "postcss"
```

Comment:
They are all valid values.

By default, JavaScript files use `babylon`, TypeScript files use `typescript`, and CSS/LESS/SASS files use `postcss`. However you could set the parser to `typescript` on JavaScript files if you really wanted to.